about.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Lato"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <title>Microverse Concert</title>
+  </head>
+  <body>
+    <section class="navigation">
+      <nav class="top-bar">
+        <i class="fa fa-facebook"></i>
+        <i class="fa fa-instagram"></i>
+        <i class="fa fa-youtube"></i>
+        <a href="#">Login</a>
+      </nav>
+      <nav class="flex-nav">
+        <div class="menu-open">
+          <img src="assets/img/ic_menu.svg" alt="" />
+        </div>
+        <div class="menu">
+          <a href="index.html" class="logo"
+            ><img src="assets/img/logo.png" alt=""
+          /></a>
+          <a href="about.html">About</a>
+          <a href="#media">Media</a>
+          <a href="#lineup">Line up</a>
+          <a href="#buytickets">Buy tickets</a>
+          <div class="menu-close">
+            <img src="assets/img/Union.svg" alt="" />
+          </div>
+        </div>
```

Comment:
- [ ] Kindly check my comments about using semantic HTML tags to structure the page.